xrviz/dashboard.py

```diff
@@ -1,7 +1,6 @@
 import panel as pn
 import xarray as xr
 import hvplot.xarray
-from cartopy import crs as ccrs
```

Comment:
This is now imported elsewhere?

Reply:
Now `cartopy` is not being imported anywhere.